.github/workflows/pr-preview.yml

```diff
@@ -14,7 +14,8 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  S3_BUCKET: ghgc-custom-interfaces-develop
+  S3_BUCKET: ghgc-dev-custom-interfaces
```

Comment:
why change the bucket?

Reply:
That bucket already has a CloudFront distribution associated with it. It seems like a good idea to have the preview environment use the same bucket as `dev` so we can catch any policy changes in S3 or CloudFront during the preview stage.